Testreglar/3.3.1/Nett/nett-3.3.1b.json

```diff
@@ -0,0 +1,256 @@
+	"id": "nett-3.3.1b",
+	"testlabId": 411,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>For skjemaelementer der inndatafeil blir oppdaget automatisk, er følgende punkter oppfylt:</p>\r\n<ul>\r\n<li>Skjemaelementet der feilen oppstod identifiseres og</li>\r\n<li>Brukeren får en presis tekstlig beskrivelse av feilen</li>\r\n<li>Feilmeldingen er kodet som tekst og</li>\r\n<li>Informasjonen blir liggende i skjemaet, med mindre informasjonen er knyttet til personvern eller sikkerhet</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "2.5"
+		},
+		{
+			"title": "2.6"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"label": "URL/Side:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har testsiden digitale skjema/skjemaelement?",
+			"ht": "<p><strong>Merk: </strong>Du skal ikke teste skjema i PDF, Word eller lignende.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke skjema."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hvilket skjema/prosess tester du?",
+			"ht": "<p>Beskriv skjema/prosess, slik at det er mulig å identifisere det i ettertid.</p>",
+			"type": "tekst",
+			"label": "Skjema/prosess:",
+			"multilinje": true,
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.4"
+				}
+			}
+		},
+		{
+			"stegnr": "2.4",
+			"spm": "Er det forhåndsutfylt informasjon i skjema?",
+			"ht": "<p>Forhåndsutfylt informasjon betyr at det allerede står inndata i skjemaet når testsiden er lastet inn.  </p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.5"
+				},
+				"nei": {
+					"type": "gaaTil",
+					"steg": "2.6"
+				}
+			}
+		},
+		{
+			"stegnr": "2.5",
+			"spm": "Er det mulig å endre forhåndsutfylt informasjon i et eller flere skjemaelementer?",
+			"ht": "<p>Sjekk om du kan endre forhåndsutfylt informasjon i et eller flere skjemaelementer.</p>\n<p><strong>Merk:</strong> Hvis du ikke får endre informasjonen i minst et av skjemaelementene som er forhåndsutfylt, skal skjemaet ikke testes.</p>",
+			"type": "jaNei",
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.6"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden krever ikke inndata i et bestemt format eller innenfor bestemte verdier."
```

Comment:
Ser ikke at dette utfallet svarer på spørsmålet? Spørsmålet spør om du får endret forhåndsutfylt informasjon, mens utfallet sier at det ikke er krav til inndataformat? To forskjellige ting?

Reply:
Endret etter din kommentar.